app/src/main/java/br/com/lucas/pomodoroapp/ui/listTaskScreen/AdapterItem.kt

```diff
@@ -0,0 +1,20 @@
+package br.com.lucas.pomodoroapp.ui.listTaskScreen
+
+data class AdapterItem(
+    val uid: Int,
+    val taskName: String,
+    val taskMinutes: Int,
+    var isSelected: Boolean = false
+){
+    fun isTaskSelected(): Boolean{
```

Comment:
Is it necessary a method because you already have the variable `isSelected`, and it is public?
Maybe u could do something like:

```
data class AdapterItem(
         val uid: Int,
         val taskName: String,
         val taskMinutes: Int,
         private var isSelected: Boolean = false
) {
       ....
       val isTaskSelected: Boolean
              get() = isSelected
}
```

Reply:
Greeeat, a val is enough! =D